src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './store/index.js';
+import App from './App.js';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+const root = document.getElementById('root');
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+ReactDOM.createRoot(root).render(
+  <Provider store={store}>
+    <React.StrictMode>
+      <App />
+    </React.StrictMode>
+  </Provider>,
```

Comment:
- [ ] There appear to be other errors relating to your App component that you need to fix.

  In conclusion, Kindly look into your app and ensure it can be set up without error and it can fetch greetings from the API backend and display the greetings without any errors too.

  ![image](https://github.com/david30-maker/hello-world-react-two/assets/36542986/2daace25-80c6-45fb-ac73-359bd3b3cf39)